scipy/optimize/_shgo_lib/_vertex.py

```diff
@@ -237,9 +238,9 @@ def __getitem__(self, x, nn=None):
         except KeyError:
             self.index += 1
             xval = self.Vertex(x, index=self.index)
-            # logging.info("New generated vertex at x = {}".format(x))
-            # NOTE: Surprisingly high performance increase if logging
-            # is commented out
+            logging.getLogger('scipy.optimize.shgo').info(
+                "New generated vertex at x = %s", x
+            )
```

Comment:
What is the performance difference? Is the old deleted comment irrelevant with the way that you set this up?

Generally, we'd do `logger = logging.getLogger('scipy.optimize.shgo')` up at the top-level of the module, not create one at each logging statement.

Reply:
Yes, I would say the old comment is irrelevant. I did some trial runs of the benchmarking suite during development, and the relative difference in execution time with and without this logging statement was less than 1% which is less than the typical variation between individual runs of the same code on my machine.